src/Apple/composer.json

```diff
@@ -34,7 +34,7 @@
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
         "ext-openssl": "*",
-        "firebase/php-jwt": "^5.2",
+        "firebase/php-jwt": "^6.0",
```

Comment:
```suggestion
        "firebase/php-jwt": "^5.2 || ^6.0",
```

Reply:
Sure, that's preferable, but there wasn't much sense in it seeing that `eveonline` only requires v6.0.